### DISCUSSION TITLE
Build via continuous integration (CI) workflow

### DISCUSSION THREAD

**Discussion by tboy1337** [masterking32/MasterHamsterKombatBot#148] [Q&A]
could you use CI to build an exe for easier usage but leave the config as something like config.config so it is editable?

https://docs.github.com/en/actions/use-cases-and-examples/building-and-testing/building-and-testing-python

**Comment by vashamamasha2014** [comment]
Bro, go to the PyCharm and write in the terminal: ```pyinstaller 
--noconsole --onefile --add-data "%USERPROFILE%\PycharmProjects\pythonProject\MasterHamsterKombatBot\utilities.py;."` --add-data "%USERPROFILE%\PycharmProjects\pythonProject\MasterHamsterKombatBot\config.py;." --add-data "%USERPROFILE%\PycharmProjects\pythonProject\MasterHamsterKombatBot\useful_files;useful_files" "%USERPROFILE%\PycharmProjects\pythonProject\MasterHamsterKombatBot\main.py"```
just substitute the path to these scripts, and you will get a .exe file to run in the background. 
![image](https://github.com/user-attachments/assets/703cc5be-bd5b-4733-a887-3fa4a541968d)

**Comment by tboy1337** [comment]
New discussion: [#303](https://github.com/masterking32/MasterHamsterKombatBot/discussions/303)
